### PR TITLE
Activating Open Collective

### PR DESCRIPTION
**Pull request by monkeywithacupcake**
Hi, I'm making updates for Open Collective. Either you or a supporter signed this repo up for Open Collective. This pull request adds backers and sponsors from your Open Collective https://opencollective.com/ant-design ❤️
  
  It adds two badges at the top to show the latest number of backers and sponsors. It also adds placeholders so that the avatar/logo of new backers/sponsors can automatically be shown without having to update your README.md. [[more info](https://github.com/opencollective/opencollective/wiki/Github-banner)]. See how it looks on [this repo](https://github.com/apex/apex#backers).
We have also added a `postinstall` script to let people know after `npm|yarn install` that you are welcoming donations (optional). [[More info](https://github.com/OpenCollective/opencollective-cli)]
You can also add a "Donate" button to your website and automatically show your backers and sponsors there with our widgets. Have a look here: https://opencollective.com/widgets

  P.S: As with any pull request, feel free to comment or suggest changes. The only thing "required" are the placeholders on the README because we believe it's important to acknowledge the people in your community that are contributing (financially or with code!).

  Thank you for your great contribution to the open source community. You are awesome! 🙌
  And welcome to the open collective community! 😊

  Come chat with us in the #opensource channel on https://slack.opencollective.com - great place to ask questions and share best practices with other open source sustainers!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -164,14 +165,19 @@
         "dom-closest": "^0.2.0",
         "dom-scroll-into-view": "^1.2.1",
         "enquire.js": "^2.1.6",
-        "json2mq": "^0.2.0",
         "is-negative-zero": "^2.0.0",
+        "json2mq": "^0.2.0",
```
Comment (tangjinzhou) [inline package.json]: What role does it play?

File: package.json [hunk 2]
```diff
@@ -36,7 +36,8 @@
         "lint": "eslint -c ./.eslintrc --fix --ext .jsx,.js,.vue ./components",
         "lint:style": "stylelint \"./examples/**/*.less\" --fix --syntax less",
         "commitmsg": "commitlint -x @commitlint/config-conventional -e $GIT_PARAMS",
-        "codecov": "codecov"
+        "codecov": "codecov",
+        "postinstall": "opencollective-postinstall"
```
Comment (tangjinzhou) [inline package.json]: `opencollective-postinstall: not found`

Is `opencollective-postinstall` missing?

**Comment by monkeywithacupcake** [comment]
You can remove the changes to package.json and still keep the readme options to donate

File: package.json (hunk 1 above)
Comment (monkeywithacupcake) [reply package.json]: Looks like it just moved down a space - you can see it on line 167 in the original version <-

**Pull request closed without merging**